apps/src/publicKeyCryptography/ModuloClock.jsx

```diff
@@ -0,0 +1,19 @@
+/** @file Centered animated Modulo Clock component for the Crypto widget levels */
```

Comment:
Wonder if there's some sweet React SVG drawing library that could be used for this.